spec/book_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Book do
+  before :each do
+    @book = Book.new('House of Terror', 'Steve Ford')
+  end
+
+  describe '#new' do
+    it 'Return a new book object' do
+      @book.should(be_an_instance_of(Book))
+    end
+
+    it 'throws an ArgumentError' do
+      -> { Book.new('title') }.should raise_exception ArgumentError
+    end
+  end
+
+  describe '#title' do
+    it 'returns the title' do
+      @book.title.should eql 'House of Terror'
+    end
+  end
+
+  describe '#author' do
+    it 'returns the author' do
+      @book.author.should eql 'Steve Ford'
+    end
+  end
```

Comment:
- [ ] @Team Great work adding tests for the book classes, however you forgot to add tests for these methods ` add_rental, book_to_json, self.from_json, save_books and load_books`, its  project requirement to add tests for all methods
![image](https://github.com/Omwomo/school-library/assets/61408860/a1e0a748-ab1a-40c5-b631-9873c106c7df)